src/pages/storage/forms/StoragePoolForm.tsx

```diff
@@ -52,6 +58,13 @@ export const storagePoolFormToPayload = (
         [getPoolKey("ceph_rbd_features")]: values.ceph_rbd_features,
         source: values.source,
       };
+    } else if (isZFSDriver) {
+      return {
+        [getPoolKey("zfs_clone_copy")]: values.zfs_clone_copy,
+        [getPoolKey("zfs_export")]: values.zfs_export,
```

Comment:
Once I set an override for any of the two, on removing the override it will not remove the override on update, but the old value will stay.

Reply:
the issue is the `updatePool` api function makes a call to the `PATCH` endpoint, when we clear configs in `getConfigurationRow` we simply set the formik field to `undefined`. In this case, the data will not be updated since it's a PATCH request, we might need to use the PUT endpoint instead but also we need to watch out for fields that cannot be updated

Reply:
This should work, ensuring we have empty string in the payload, when the value is undefined:
```
        [getPoolKey("zfs_clone_copy")]: values.zfs_clone_copy ?? "",
```

Reply:
Does empty string become defaults in the backend?

Reply:
just tested, that works :astonished: